setup.py

```diff
@@ -29,6 +29,8 @@ def requires(filename: str):
     packages=find_packages(exclude=excludes),
     install_requires=requires("requirements.txt"),
     extras_require={"dev": requires("dev-requirements.txt")},
+    package_data={"mmpy_bot": ["mmpy_bot/version.txt"]},
+    include_package_data=True,
```

Comment:
Is this necessary? I'm pretty sure the manifest already covers this?

Reply:
The Manifest only covers inclusion in the source tarball.
If you don't include these lines the file will not be installed.
That was part of the reason why `version.txt` wasn't being installed even when outside the `mmpy_bot` folder.